microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -0,0 +1,44 @@
+Any WebView2 data collection complies with the laws, regulations, and best practices for privacy, and WebView2 has mechanisms to ensure privacy.
+
+
+<!-- ====================================================================== -->
+## Optional diagnostic data
+
+WebView2 is considered a Windows component, and the data collection consent is governed by the Windows **Settings > Privacy & security > Diagnostics & feedback > Diagnostic data** setting on Windows 10 and 11 as a centralized switch.
+Users can control the data collection of WebView2 through the Windows **Diagnostic data** setting on Windows 10 and 11. As a developer, you are welcome to convey this to your users and invite them to use the Windows **Diagnostic data** setting to fit their preference.
+WebView2 developers have no control over the Windows **Diagnostic data** setting. This is similar to the Windows APIs.
+
+When the Windows **Diagnostic data** setting is turned **On**, WebView2-related data including API usage, SDK usage, and creation failure is collected. When this setting is **On**, browser diagnostic data is collected; however, personalization data is not collected. Only browser events that are relevant to WebView2 are collected. Regardless of the Windows **Diagnostic data** setting, WebView2 collects required data that's necessary to maintain performance and reliability; see [Diagnostics, feedback, and privacy in Windows](https://support.microsoft.com/en-us/windows/diagnostics-feedback-and-privacy-in-windows-28808a2b-a31b-dd73-dcd3-4559a5199319).
+
+There are two APIs that exist to give developers more control over data collection: [Disable SmartScreen](https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/specs/IsSmartScreenRequired.md) and [Custom Crash Reports](https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/specs/DisableCrashReporting.md). See [Security and privacy](../concepts/overview-features-apis.md#security-and-privacy) in _Overview of WebView2 features and APIs_.
+
+
+<!-- ====================================================================== -->
+## SmartScreen
+
+Microsoft Defender SmartScreen ("SmartScreen") is enabled by default. If you don't disable SmartScreen, you must provide notice to all users that your software includes Microsoft Defender SmartScreen and collects and sends the user's information to Microsoft as disclosed in [Microsoft Privacy Statement](https://aka.ms/privacy) and the [Microsoft Edge Privacy Whitepaper](../../privacy-whitepaper/index.md#smartscreen). Generally, all other services in `edge://settings/privacy` are turned off, for WebView2.
+
+The managing SmartScreen API includes the `IsReputationCheckingRequired` property, which controls whether SmartScreen is enabled.  See [Security and privacy](../concepts/overview-features-apis.md#smartscreen) in _Overview of WebView2 features and APIs_.

```

Comment:
```suggestion
The managing SmartScreen API includes the `IsReputationCheckingRequired` property, which controls whether SmartScreen is enabled.  See [SmartScreen](../concepts/overview-features-apis.md#smartscreen) in _Overview of WebView2 features and APIs_.
```
update link text
note that the See Also section below has a broader link; it links to the parent h4 section